src/main/scala/cromwell/binding/command/Command.scala

```diff
@@ -131,7 +131,7 @@ case class Command(parts: Seq[CommandPart]) {
     val trimmed = stripAll(s, "\r\n", "\r\n \t")
     val parts = trimmed.split("[\\r\\n]+")
     val indent = parts.map(leadingWhitespaceCount).min
-    parts.map(_.substring(indent)).mkString("")
+    parts.map(_.substring(indent)).mkString("\n")
```

Comment:
You're probably right, there are probably cases where that does matter.  I can't think of a great easy solution at the moment, but maybe because it's late at night.  Well right now it's *really* broken and this would only make it 'edge-case' broken